openedx/core/djangoapps/heartbeat/tasks.py

```diff
@@ -4,6 +4,6 @@
 from celery.task import task
 
 
-@task()
+@task(routing_key=settings.HIGH_PRIORITY_QUEUE)
```

Comment:
Most of the tasks with different routing keys have a custom setting so that the routing key can be overridden.
For instance the [`BULK_ROUTING_KEY`](https://github.com/edx/edx-platform/blob/441d6384eef5ed0b62e936a7d85958d10e43d92e/lms/envs/common.py#L1947) has it's own setting.

Could you set this up similarly, instead of the direct assignment to the high priority queue?